front/components/assistant/conversation/ConversationMessage.tsx

```diff
@@ -0,0 +1,24 @@
+import { Avatar } from "@dust-tt/sparkle";
+
+/**
+ * Parent component for both UserMessage and AgentMessage, to ensure avatar,
+ * side buttons and spacing are consistent between the two
+ */
+export function ConversationMessage({
+  avatarVisual,
+  children,
+}: {
+  avatarVisual?: string | React.ReactNode;
```

Comment:
Name is not great. Why a node? This seems to mean your interface is not the right one?